tcms/testplans/templates/testplans/mutable.html

```diff
@@ -0,0 +1,100 @@
+                    <label for="id_version">{% trans "Version" %}</label>
+                    <a href="{% url 'admin:management_version_add' %}?_popup"
+                       id="add_id_version" alt="{% trans 'add new Version' %}" title="{% trans 'add new Version' %}">
+                        +
+                    </a>
+                </div>
+                <div class="col-md-3 col-lg-3">
+                    <select id="id_version" name="version" class="form-control">
+                        {% for product_version in form.product_version.field.queryset %}
+                            <option value="{{ product_version.pk }}">
+                                {{ product_version.value }}
+                            </option>
+                        {% endfor %}
+                    </select>
+                </div>
+                <div class="col-md-1 col-lg-1">
+                    <label for="id_type">{% trans "Type" %}</label>
+                </div>
+                <div class="col-md-3 col-lg-3">
+                    <select class="form-control" id="id_type" name="type">
+                        {% for type in form.type.field.queryset %}
+                            <option value="{{ type.pk }}"> {{ type }} </option>
+                        {% endfor %}
+                    </select>
+                </div>
+            </div>
+
+            <div class="form-group">
+                <label class="col-md-1 col-lg-1" for="id_parent_id">{% trans "Parent ID" %}</label>
+                <div class="col-md-3 col-lg-3">
+                    <input type="text" id="id_parent_id" name="parent_id" value="" class="form-control">
+                </div>
+                <label class="col-md-1 col-lg-1" for="id_product">{% trans "Reference link" %}</label>
+                <div class="col-md-7 col-lg-7">
+                    <input type="text" id="id_summary" name="summary" value="" class="form-control">
+                </div>
+            </div>
+
+            <div class="form-group">
+                <label class="col-md-12 col-lg-12" for="messageArea" for="id_summary"> {% trans "Test plan document:" %}</label>
+                <div class="col-lg-12 col-md-12">
+                    <div>{{ form.text }}</div>
+                </div>
+            </div>
+
+            <button type="submit" class="btn btn-default btn-lg">{% trans "Save" %}</button>
+
+        </form>
+    </div>
+
+<script src="{% static "grappelli/jquery/jquery.min.js" %}" type="text/javascript"></script>
+<script src="{% static "grappelli/js/grappelli.min.js" %}" type="text/javascript"></script>
+<script src="{% static "admin/js/admin/RelatedObjectLookups.js" %}" type="text/javascript"></script>
+
+<script src="{% static 'bootstrap-select/dist/js/bootstrap-select.min.js' %}" type="text/javascript"></script>
+    <script src="{% static 'js/jsonrpc.js' %}"></script>
+
+{#<script src="{% static 'testplans/js/mutable.js' %}" type="text/javascript"></script>#}
```

Comment:
Pretty sure none of these js files are used ATM so whatever we need must be added in a separate commit.